docs/index.md

```diff
@@ -763,9 +763,9 @@ Mocha supports the `err.expected` and `err.actual` properties of any thrown `Ass
     -w, --watch                             watch files for changes
     --check-leaks                           check for global variable leaks
     --full-trace                            display the full stack trace
-    --compilers <ext>:<module>,...          use the given module(s) to compile files (default: )
+    --compilers <ext>:<module>,...          use the given module(s) to compile files
```

Comment:
So for all of these README changes, why did you remove the default text altogether rather than just adding '[]' where it was missing?